vision_msgs/vision_msgs/create_aabb.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VISION_MSGS__CREATE_AABB_HPP_
+#define VISION_MSGS__CREATE_AABB_HPP_
+
+#include "vision_msgs/msg/bounding_box2_d.hpp"
+#include "vision_msgs/msg/bounding_box3_d.hpp"
+
+namespace vision_msgs
+{
+/**
+ * Create an axis-aligned bounding box (AABB) given the upper-left corner,
+ * width, and height. This allows easy conversion from the OpenCV rectangle
+ * representation.
+ */
+static inline msg::BoundingBox2D createAABB2D(
```

Comment:
Is the static really necessary? That makes this have threading issues if called in multiple threads at high frequency in a processing pipeline

Reply:
ditto below. Inline should suffice

Reply:
I did not touch those files. They had the static already.